app/code/Magento/UrlRewriteGraphQl/etc/schema.graphqls

```diff
@@ -26,3 +27,13 @@ type HttpQueryParameter @doc(description: "The object details of target path par
     name: String @doc(description: "Parameter name")
     value: String @doc(description: "Parameter value")
 }
+
+interface RoutableInterface @typeResolver(class: "Magento\\UrlRewriteGraphQl\\Model\\RoutableInterfaceTypeResolver") @doc(description: "Description here.") {
+    display_metadata: DisplayMetadata @doc(description: "Display Meta-Data details") @resolver(class: "Magento\\UrlRewriteGraphQl\\Model\\Resolver\\DisplayMetaData")
```

Comment:
I stopped here, because I don't understand the schema well enough to offer any suggestions. This area of the schema is not intuitive, and the descriptions need to be more than a restatement of the attribute/type names.

I would also prefer that "metadata" not be used as part of the attribute/data type name, but changing that would require approval from Magento developers/architects.

Reply:
Hi @keharper ... we had a call with @cpartica earlier regarding returning the metadata response as object. I hope this is what expected.